### ISSUE TITLE
Are you still working on this? Any plans to update it to Vue 3?

### ISSUE THREAD

**Issue by majortom64** [tomgobich/amplify-thumbtemps#1]
It seems like the best Cognito Auth example I have found, but I am having some trouble moving it to Vue 3.

**Comment by tomgobich** [comment]
No, I don't have any intentions of continuing work on this project in particular. I have code in this project that tweaks the router a bit, adding layout and middleware support. If I had to guess that's where the bulk of your troubles are coming from. You'll probably stand a better chance using the Vue CLI to create a new Vue 3 project then plucking out the Cognito code from this project. 

I've been making Nuxt + Amplify videos, including auth. I'll probably cover Vue 3 in the future as well. If I do I'll let you know, the approach will be very similar to this repo.

Current Nuxt + Amplify videos: https://www.youtube.com/playlist?list=PL9dIWiKCV573u7FUcj6bF1vwunYFEDA6Y

**Comment by majortom64** [comment]
Thanks. I did get it working (I had started with a clean Vue project, but mostly what I really needed was not just your Amplify/Cognito code, but your Form handling stuff). I watched your Nuxt one (which was great), which is how I got here. :-) I love Nuxt, but I really want some of the Vue 3 features, so I am stuck for the moment.

Currently trying to understand how to use route guards to require auth for various routes.

Your Nuxt/Amplify videos were a big help in getting me through this process. :-)

I would love to know if you do a Vue 3 version of them.
